src/components/change-wallet/AddressRow.js

```diff
@@ -202,7 +203,9 @@ export default function AddressRow({
               <StyledBottomRowText
                 color={colors.alpha(colors.blueGreyDark, 0.5)}
               >
-                {cleanedUpBalance || 0} ETH
+                {lang.t('wallet.change_wallet.balance_eth', {
```

Comment:
is this needed? `ETH` will always be `ETH` regardless of language correct?

Reply:
@jinchung Ah, that makes sense. Since `ETH` will be the same I reverted back to the original way for this and the other instance where this happened.